user-guide/Reference/faq/General_configuration.md

```diff
@@ -60,17 +60,29 @@ The time server will provide the correct time settings for all DMAs.
 
 To turn any computer into a time server, whether it serves as a DMA or not, do the following.
 
+> [!IMPORTANT]
+> Below information applies to Windows Server 2008 R2, Windows 7 and later.
+
 1. Log on locally to the machine that will act as time server, or establish a remote desktop session with it.
 
-1. Open Registry Editor (*Start \> Run... \> regedit*).
+1. Stop the "Windows Time" service via (*Start \> Control Panel \> System and Security \> Administrative Tools \> Services*).
+
+1. Open the registry editor via (*Start \> Run... \> regedit*).
+
+1. Search for the "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\W32Time\TimeProviders\NtpServer".
```

Comment:
We usually place literal references to something in the UI in italics instead of in quotation marks, so this should be "1. Search for *HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\W32Time\TimeProviders\NtpServer*."